pulsar-broker-auth-sasl/src/main/java/org/apache/pulsar/broker/authentication/AuthenticationProviderSasl.java

```diff
@@ -109,4 +128,160 @@ public AuthenticationState newAuthState(AuthData authData,
             throw new AuthenticationException(t.getMessage());
         }
     }
+
+    // for http auth.
+    private static final long LIVE_SECONDS = 3600;
```

Comment:
`LIVE_SECONDS` is a generic word. How about calling it SASL_ROLE_TOKEN_LIVE_SECONDS ?